### ISSUE TITLE
Car Parts

### ISSUE THREAD

**Issue by mrelliot69** [sunrise-php/vin#80]
Is it possible to also get the car parts from the vin?
and other properties like fuel type,...

**Comment by Hayleene** [comment]
Hey, is it possible to get car racing parts like racing shoes inside U.S?

**Comment by Lucassane** [comment]
Have you searched on Google about car racing parts? I think there are multiple options available.

**Comment by MikeColady** [comment]
> Hey, is it possible to get car racing parts like racing shoes inside U.S?

Yes, you can easily find car racing parts like racing shoes within the U.S. Several reputable retailers offer a wide range of racing gear and accessories, including brands like Alpinestars, Sparco, and OMP. Check out stores like Club Racers Garage for a comprehensive selection of these [motor racing shoes](https://clubracersgarage.com/collections/driving-shoes). They provide online shopping options and shipping across the U.S., making it convenient to find the gear you need.
